Lib/test/test_random.py

```diff
@@ -806,6 +806,20 @@ def test_getrandbits(self):
         self.assertEqual(self.gen.getrandbits(100),
                          97904845777343510404718956115)
 
+    def test_getrandbits_2G_bits(self):
```

Comment:
Shouldn't this be decorated with bigmemtest?  Ditto the second test.

Reply:
It requires only 800 MB of memory. Too small for bigmemtest. The second test requires about 1300 MB of memory. This may be too small too, but I'm going to add bigmemtest after testing on 32-bit buildbots.

Reply:
> It requires only 800 MB of memory. Too small for bigmemtest.

Where is the boundary?  IIRC, memory limit is configurable.

Reply:
It was 1GiB (you could not specify less than 1 GiB for -M option in regrtest). But now it seems is 2 GiB.

Reply:
Actually, it was initially 2.5 GiB, then lowered to 2 GiB. But I add bigmemtest even for some tests that need 1-2 GiB.